doc/releasenotes/8_0_0_release_notes.md

```diff
@@ -0,0 +1,274 @@
+* VDiff: fix panic for tables with a unicode_loose_md5 vindex #6640
+* Vtctld UI: Add mysqld port display back to vtctld web UI #6598 #6597 
+* Vtctld: Fix workflow CopyState fetching (and thus printing). #6657
+* Fix unsaved event regression #6710
+* Change tracker to use allprivs instead of dba #6720
+* Fixes long wait filter keyspace #6721
+* s3 ListBackups doesn’t work from root directory #6732
+* Vtctld: ExecuteFetchAsDba and ExecuteFetchAsAllPrivs do not escape db names. #6545
+* Can't switch on restore_from_backup on existing shard with no tables #4896
+* Refactor conn.go #6818
+
+
+## Functionality Added or Changed
+
+### VTGate / MySQL compatibility
+
+* Add support for DELETE IGNORE #6722
+* Only take on simple dual queries in the vtgate #6666
+* Support mycli access to vtgate #4365
+* Add support for unicode_loose_xxhash #6457
+* Provide virtual information_schema #5394
+* information_schema.tables_schema comparison with dynamic value no longer works #6827
+* Mysqldump: Allow setting multiple variables in one statement #5401
+* Block lock function usage other than dual table #6470
+* Lock Session Support #6517
+* Make sure lookup vindexes are queryable inside transaction #6499
+* Add unicode_loose_xxhash Vindex type #6549
+* Support vindex update or delete when destination is provided by client in dml query #6554
+* Make sure to handle EXPLAIN on tablets #6579
+* Rewrite SHOW TABLES #6615
+* Make offset work in OLAP mode #6655
+* Support sql_calc_found_rows with limit in sharded keyspace #6680
+* Lock shard session heartbeat #6683
+* Mysql -ssl tag is deprecated in the later versions of mysql #6682
+* Use statement support in olap mode #6692
+* Delete row using consistent lookup vindex in where clause #6700
+* Add vtexplain support for non-evenly sharded keyspaces #6705
+* Make sure dual queries set found_rows correctly #6718
+* Add if not exists support to database ddl in parsing #6724
+* Allow Update of same vindex using same vindex lookup on consistent lookup vindex #6736
+* Update Vindex only on changes #6737
+* Add support for `SHOW (DATABASES|VITESS_SHARDS|VITESS_TABLETS) LIKE` #6750
+* 'CASCADE' support #6743
+* Mask database name #6791
+* Make sure to backtick schema names properly #6550
+* Transaction Limiter: fix quota leak if Begin() errors or if the tx killer runs #6731
+* Allow Load Data From S3 #6823
+
+### OLAP Functioanlity 
+* Allow switching between OLAP and OLTP #6691
+* Use statement support in olap mode #6692
+* Fix error handling in olap mode #6940
+
+
+### Set Statement Support
+Set statement support is added in Vitess. There are [some system variables](https://github.com/vitessio/vitess/blob/master/go/vt/sysvars/sysvars.go#L147,L190) which are disabled by default and can be enabled using flag `-enable_system_settings` on VTGate. These system variables are set on the mysql and they does not use connection pool and have their own dedicated connection.
```

Comment:
`These system variables are set on the mysql and they does not use connection pool and have their own dedicated connection.`

=>

`These system variables are set on the backing MySQL instance, and will force the connection to be dedicated instead of part of the connection pool`

Reply:
Updated